src/interface.jl

```diff
@@ -19,7 +19,7 @@ export hasdensity
 @inline hasdensity(::Any) = false
 
 function check_hasdensity(d)
-    hasdensity(d) || throw("Object of type $(typeof(d)) is not compatible with DensityInterface")
+    hasdensity(d) || error("Object of type $(typeof(d)) is not compatible with DensityInterface")
```

Comment:
```suggestion
    hasdensity(d) || throw(ArgumentError("Object of type $(typeof(d)) is not compatible with DensityInterface"))
```

Could you also add a `@test_throws ArgumentError logdensityof("foo")` to the tests?